main.go

```diff
@@ -84,4 +84,32 @@ func main() {
 		handleError(err)
 	}
 	fmt.Println("Number of transactions:", len(transactionsResp.Transactions))
+
+	// POST /payment_initiation/recipient/create
+	paymentRecipientCreateResp, err := client.CreatePaymentRecipient("John Doe", "GB33BUKB20201555555555", plaid.PaymentRecipientAddress{
+		Street:     []string{"Street Name 999"},
+		City:       "City",
+		PostalCode: "99999",
+		Country:    "GB",
+	})
+	handleError(err)
+	fmt.Println("Recipient ID:", paymentRecipientCreateResp.RecipientID)
+
+	// POST /payment_initiation/recipient/get
+	paymentRecipientGetResp, err := client.GetPaymentRecipient(paymentRecipientCreateResp.RecipientID)
+	handleError(err)
+	fmt.Println("Recipient get response:", paymentRecipientGetResp)
+
+	// POST /payment_initiation/payment/create
```

Comment:
Need to add /payment_initiation/payment/token/create after this call.

Reply:
https://github.com/plaid/plaid-go/pull/89/commits/8eb0eb526de6faeeee421cb50c9bad838293a711

Reply:
Leave reply